projects/js-packages/components/components/button/index.tsx

```diff
@@ -49,7 +50,15 @@ const Button = forwardRef< HTMLInputElement, ButtonProps >( ( props, ref ) => {
 
 	const externalIconSize = size === 'normal' ? 20 : 16;
 	const externalIcon = isExternalLink && (
-		<Icon size={ externalIconSize } icon={ external } className={ styles[ 'external-icon' ] } />
+		<>
+			<Icon size={ externalIconSize } icon={ external } className={ styles[ 'external-icon' ] } />
+			<VisuallyHidden as="span">
```

Comment:
I was a bit hesitant if we should add Translations and text in the core button, but it seems that this approach they use in core wordpress. I think it ok to have it here, especially if we want to make all external links accessible.

Reply:
I think it's okay too. The text is generic enough, and I'd say it's not far-fetched to assume that the codebases using this package already import `__`.